shell/platform/android/io/flutter/embedding/engine/FlutterJNI.java

```diff
@@ -230,14 +230,28 @@ public boolean getIsSoftwareRenderingEnabled() {
   }
 
   /**
-   * Observatory URI for the VM instance.
+   * VM Service URI for the VM instance.
    *
    * <p>Its value is set by the native engine once {@link #init(Context, String[], String, String,
    * String, long)} is run.
    */
   @Nullable
+  public static String getVMServiceUri() {
+    return vmServiceUri;
+  }
+
+  /**
+   * VM Service URI for the VM instance.
+   *
+   * <p>Its value is set by the native engine once {@link #init(Context, String[], String, String,
+   * String, long)} is run.
+   *
+   * @deprecated replaced by {@link #getVMServiceUri()}.
+   */
+  @Deprecated
+  @Nullable
   public static String getObservatoryUri() {
```

Comment:
@jmagman renaming this method had broken the espresso plugin.

Reply:
Interesting... https://github.com/flutter/plugins/blob/7686be7ecbe1cc049423715b5e2d897cb63a43c1/packages/espresso/android/src/main/java/androidx/test/espresso/flutter/action/FlutterViewAction.java#L108